BricksAndHearts.UnitTests/ControllerTests/Landlord/LandlordControllerTests.cs

```diff
@@ -21,4 +28,19 @@ public void RegisterGet_CalledByUnregisteredUser_ReturnsRegisterViewWithEmail()
         result!.Model.Should().BeOfType<LandlordProfileModel>()
             .Which.Email.Should().Be(unregisteredUser.GoogleEmail);
     }
+
+    [Fact]
+    public async void ApproveCharter_CallsApproveLandlord()

```

Comment:
```suggestion
    public async void ApproveCharter_CalledWithAdminPermissions_CallsApproveLandlord()
```

Don't forget to test this endpoint without admin permissions

Reply:
I looked into unit testing things which should require authorisation, and from what I could find it's not actually possible - the unit test calls the method and ignores any [Authorise] labels above it. The only way of checking authorisation is to write a test that literally just checks that any relevant method/endpoint for the [Authorise] label, and I'm not sure if this is worth doing - if it is, it's probably worth raising a separate ticket as it would be a shared test, listing anything which requires authorisation, rather than specific to this endpoint.